web/src/emoji_picker.ts

```diff
@@ -243,24 +266,45 @@ function filter_emojis() {
             if (category.name === "Popular") {
                 continue;
             }
+
             const emojis = category.emojis;
             for (const emoji_dict of emojis) {
                 for (const alias of emoji_dict.aliases) {
                     const match = search_terms.every((search_term) => alias.includes(search_term));
                     if (match) {
-                        search_results.push({...emoji_dict, emoji_name: alias});
+                        if (emoji_dict.is_realm_emoji) {
+                            search_results.push({
+                                ...emoji_dict,
+                                emoji_name: alias,
+                                reaction_type: "realm_emoji",
+                                is_realm_emoji: true,
```

Comment:
Why are these changes required? The `emoiji_dict` object already has those fields.